packages/plugins/other/visitor-plugin-common/src/base-resolvers-visitor.ts

```diff
@@ -972,14 +972,15 @@ export class BaseResolversVisitor<
 
       const resolverType = isSubscriptionType ? 'SubscriptionResolver' : directiveMappings[0] ?? 'Resolver';
 
+      const avoidOptionals = this.config.avoidOptionals?.object ?? this.config.avoidOptionals === true;
```

Comment:
Something is wrong with the types here, because it thinks it can only be `AvoidOptionalsConfig` but it can apparently also still be a `boolean` per `ts-resolvers.spec.ts`.